packages/@headlessui-react/src/components/listbox/listbox.tsx

```diff
@@ -801,14 +385,17 @@ function ButtonFn<TTag extends ElementType = typeof DEFAULT_BUTTON_TAG>(
       case Keys.Space:
       case Keys.ArrowDown:
         event.preventDefault()
-        flushSync(() => actions.openListbox())
-        if (!data.value) actions.goToOption(Focus.First)
+        flushSync(() => machine.actions.openListbox())
+        if (!data.value)
+          machine.actions.goToOption({
+            focus: Focus.First,
+          })
```

Comment:
Now that this wraps I'd probably add braces

Reply:
I think I can make it not wrap. Good eye!